### PR TITLE
New comparators: LMV7219 and LTC6754 family

### PR DESCRIPTION
**Pull request by polwel**
Added new comparators:

- LMV7219 ([datasheet](https://www.ti.com/lit/ds/symlink/lmv7219.pdf))
  Screenshot_2

- LTC6754 variants ([datasheet](https://www.analog.com/media/en/technical-documentation/data-sheets/6754f.pdf))
Screenshot_1
Screenshot_2

### CODE REVIEW

**Comment by polwel** [comment]
I have made the symbol smaller, but there is only so much I can do. Now the labels for the inputs and the supply clip a bit.

image

File: Comparator.dcm [hunk 1]
```diff
@@ -90,10 +90,16 @@ K dual comparator
...
-D Single Low-Power High-Speed Push-Pull Output Comparator, SOT-23-5/SC-70
+$CMP LMV7219M5
+D Single Low-Power High-Speed Push-Pull Output Comparator, SOT-23-5
 K single comparator
-F http://www.ti.com/lit/ds/symlink/lmv7219.pdf
+F https://www.ti.com/lit/ds/symlink/lmv7219.pdf
+$ENDCMP
+#
+$CMP LMV7219M7
+D Low-Power High-Speed Push-Pull Output Comparator, SC-70-5
```
Comment (chschlue) [inline Comparator.dcm]: Prepend `Single` for consistency

File: Comparator.dcm [hunk 2]
```diff
@@ -138,27 +144,15 @@ K comparator latch
...
 #
-$CMP LTC6754HSC6
-D High Speed Rail-to-Rail Input Comparator with LVDS Compatible Outputs
-K comparator high speed lvds
-F https://www.analog.com/media/en/technical-documentation/data-sheets/6754f.pdf
-$ENDCMP
-#
-$CMP LTC6754HUD
+$CMP LTC6754xSC6
 D High Speed Rail-to-Rail Input Comparator with LVDS Compatible Outputs
```
Comment (chschlue) [inline Comparator.dcm]: Append package

File: Comparator.dcm [hunk 3]
```diff
@@ -138,27 +144,15 @@ K comparator latch
...
 K comparator high speed lvds
 F https://www.analog.com/media/en/technical-documentation/data-sheets/6754f.pdf
 $ENDCMP
 #
-$CMP LTC6754ISC6
+$CMP LTC6754xUD
 D High Speed Rail-to-Rail Input Comparator with LVDS Compatible Outputs
```
Comment (chschlue) [inline Comparator.dcm]: Append package

File: Comparator.dcm [hunk 4]
```diff
@@ -138,27 +144,15 @@ K comparator latch
...
-K comparator high speed lvds
-F https://www.analog.com/media/en/technical-documentation/data-sheets/6754f.pdf
-$ENDCMP
-#
-$CMP LTC6754IUD
-D High Speed Rail-to-Rail Input Comparator with LVDS Compatible Outputs
-K comparator high speed lvds
+K comparator high speed lvds QFN-16
```
Comment (chschlue) [inline Comparator.dcm]: Remove package

File: Comparator.lib [hunk 5]
```diff
@@ -313,55 +292,53 @@ X ~Q 8 300 100 100 L 50 50 1 1 O
...
+DEF LTC6754xUD U 0 20 Y Y 1 F N
+F0 "U" 500 -550 50 H V C CNN
+F1 "LTC6754xUD" 700 -400 50 H V C CNN
 F2 "Package_DFN_QFN:QFN-12-1EP_3x3mm_P0.5mm_EP1.65x1.65mm" 350 -600 50 H I C CNN
-F3 "" 0 0 50 H I C CNN
-ALIAS LTC6754IUD
+F3 "" -100 0 50 H I C CNN
 $FPLIST
- *QFN*
+ QFN*3x3mm*P0.5mm*
```
Comment (chschlue) [inline Comparator.lib]: `QFN*1EP*3x3mm*P0.5mm*`

File: Comparator.dcm (hunk 1 above)
Comment (polwel) [reply Comparator.dcm]: Huh, the aliased component (MCP6561) was also missing this. I added it to that one too.

Btw, do you want me to merge, rebase or even squash my changes?

Comment (chschlue) [reply Comparator.dcm]: No! Please don't do anything altering git history during a review.

**Comment by chschlue** [comment]
Thanks.

**Comment by polwel** [comment]
Nice. Thank you too for your patience!

**Pull request merged**